volumes-web-app/rockcraft.yaml

```
@@ -1,120 +1,133 @@
+# Based on https://github.com/kubeflow/kubeflow/blob/v1.9.0/components/crud-web-apps/volumes/Dockerfile
 name: volumes-web-app
 summary: Volumes Web App
 description: |
   This web app is responsible for allowing the user to manipulate PVCs in their Kubeflow
   cluster. To achieve this it provides a user friendly way to handle the lifecycle of PVC
   objects.
-version: v1.7.0_1
+version: v1.9.0
 license: Apache-2.0
-base: ubuntu:20.04
+base: ubuntu@20.04
```

```suggestion
base: ubuntu@22.04
```
for 1.9 rocks, we are using base 22.04